src/renderer/src/pass/forward.rs

```diff
@@ -454,45 +448,40 @@ impl<R> Pass<R> for DrawShaded<R> where R: gfx::Resources {
 
         // Draw every entity
         for e in &scene.fragments {
-            encoder.update_constant_buffer(
-                &self.vertex,
-                &VertexArgs {
-                    proj: scene.camera.projection,
-                    view: scene.camera.view,
-                    model: e.transform,
-                }
-            );
-
-            encoder.update_constant_buffer(
-                &self.fragment,
-                &FragmentArgs {
-                    point_light_count: point_lights.len() as i32,
-                    directional_light_count: directional_lights.len() as i32,
-                }
-            );
+            encoder.update_constant_buffer(&self.vertex,
+                                           &VertexArgs {
+                                               proj: scene.camera.proj,
+                                               view: scene.camera.view,
+                                               model: e.transform,
+                                           });
+
+            encoder.update_constant_buffer(&self.fragment,
+                                           &FragmentArgs {
+                                               point_light_count: point_lights.len() as i32,
+                                               directional_light_count: directional_lights.len() as
+                                                                        i32,
```

Comment:
Sorry, I just meant the fact that since it's slightly over the max line length, there's just a `i32,` floating by itself. At any rate yeah, `usize` would definitely make more sense here, since it doesn't make any sense to have a negative count of lights.